code/modules/surgery/surgery_step.dm

```diff
@@ -60,14 +60,27 @@
 /datum/surgery_step/proc/initiate(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
 	surgery.step_in_progress = TRUE
 	var/speed_mod = 1
+	switch(user.mind.surgery_skill)
+		if(0)
+			testing("Level 0")
+		if(1)
+			speed_mod *= 0.9
+			testing("Level 1")
+		if(2)
+			speed_mod *= 0.85
+			testing("Level 2")
+		if(3)
+			speed_mod *= 0.8
+			testing("Level 3")
+

```

Comment:
this PR: https://github.com/tgstation/tgstation/pull/46640
has big changes to how speed is used, once that goes through you will want to update changes in this file